### ISSUE TITLE
Function to generate multiple data definitions of the same type

### ISSUE THREAD

**Issue by kgoldfeld** [kgoldfeld/simstudy#125]
If I want a definition table that will generate a large number of similar variables (such as *g1*, ..., *g4* below), the only way to currently do this is by calling `defData` for each instance. If they are truly identical, I'd like to make it possible with a single call.

```r
   varname     formula variance        dist     link
1:      g1 1/3;1/3;1/3        0 categorical identity
2:      g2 1/3;1/3;1/3        0 categorical identity
3:      g3 1/3;1/3;1/3        0 categorical identity
4:      g4 1/3;1/3;1/3        0 categorical identity
5:       a         1;1        0   trtAssign identity
6:       y        0.10        0      binary identity
```

**Comment by kgoldfeld** [comment]
@assignUser I created functions `defRepeat` and `defRepeatAdd` in branch def-data-enhancements. I took a look at creating a test, but not entirely clear one would be appropriate here since the functions are largely derivative of `defData` and `defDataAdd`, respectively. I couldn't find a unit test for `defData` or `defDataAdd`, which I was hoping would serve as a model. If you have a chance, take a quick look, and let me know what you think.
